src/Sentry.Protocol/BaseScope.cs

```diff
@@ -141,7 +141,7 @@ public User User
         /// An arbitrary mapping of additional metadata to store with the event.
         /// </summary>
         public
-#if NET45
+#if NET451
```

Comment:
Next versio will be 3.0 so feel free to add breaking changes although we should keep track of them.

On a separate PR we can drop anything older than net461

Reply:
I'm stupid, we only need to change this in tests, not on the actual project. Will update.